src/Chapter2.hs

```diff
@@ -620,7 +625,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven (x : _ : xs) = x : takeEven xs
```

Comment:
Wow, you've already mastered pattern-matching and recursive functions in Haskell! :brain:

Reply:
and I increased it now with recursive go the func `takeEvenGo`